ddp-workspace/projects/ddp-sdk/src/lib/components/user/activities/userActivities.component.ts

```diff
@@ -165,8 +166,8 @@ import { tap, mergeMap } from 'rxjs/operators';
 })
 export class UserActivitiesComponent implements OnInit, OnDestroy, OnChanges, AfterContentInit {
   @Input() studyGuid: string;
+  @Input() displayedColumns: Array<DashboardColumns> = ['name', 'summary', 'date', 'status', 'actions'];
   @Output() open: EventEmitter<string> = new EventEmitter();
-  public displayedColumns = ['name', 'summary', 'date', 'status', 'actions'];
   public dataSource: UserActivitiesDataSource;
   public loaded: boolean;
   private states: Array<ActivityInstanceState> | null;
```

Comment:
I am getting a deprecation warning on line 200 around the use of mergeMap. I think we have a bunch of those rxjs warning. right now in code, but since this one is brand new, if easy to get rid of warning...

Reply:
Fixed, could you re-approve?